### DISCUSSION TITLE
Update to ubuntu-latest causing issues pulling containers from GHCR

### DISCUSSION THREAD

**Discussion by jsmilkstein** [actions/virtual-environments#6062] [General questions]
TLDR: We use `ubuntu-latest` for all our runner images, and as of Monday 8/8, the image pulls of containers we published to ghcr are erroring with `no matching manifest for linux/amd64 in the manifest list entries`.

More details: These are containers using a couple different base images, constructed with docker compose, and pushed to GHCR, all as part of a workflow. I can verify the container is actually listed under artifacts.  A subsequent workflow attempts to pull the image to build a test environment, and fails with the above message.  Both parts of the workflow are using ubuntu-latest as the runner. I tried setting them back to 18.04 and it works, and ubuntu-latest worked up until the latest update (started failing on 8/8/2022).
